pkg/kubelet/kubelet_node_status.go

```diff
@@ -1100,6 +1116,18 @@ func (kl *Kubelet) defaultNodeStatusFuncs() []func(*v1.Node) error {
 	}
 }
 
+// getNodeReadyLastHeartbeatTime get the LastHeartbeatTime on NodeReady condition.
```

Comment:
doc it returns nil if the condition doesn't exist

Reply:
ok